airbyte-integrations/bases/standard-source-test/src/main/java/io/airbyte/integrations/standardtest/source/StandardSourceTest.java

```diff
@@ -118,7 +118,8 @@
       "airbyte/source-gitlab-singer",
       "airbyte/source-google-workspace-admin-reports",
       "airbyte/source-zendesk-talk",
-      "airbyte/source-quickbooks-singer");
+      "airbyte/source-quickbooks-singer",
+      "airbyte/source-jira");
```

Comment:
if we are using new test script then we don't need this or the build block in gradle